docs/documentation/getting-started.md

```diff
@@ -168,11 +168,11 @@ in the Saros instances of your invited contacts (with synchronizing files, copyi
 {% alert info %}
 ##### Current restrictions
 
-**Number of Participants**
+###### **Number of Participants**
```

Comment:
Are the `**` still necessary?

Reply:
`h6` headings are no longer bold, making it very hard to distinguish between the text and the heading.

Reply:
In order to format all `h6` headings bold you could add
```css
h6 {
  font-weight: bold;
}
```
to `_sass/style.scss`

Reply:
This is a general style question. If we set it for this heading, we should also consider setting it for other headings as it would look weird in comparison otherwise. As we haven't used h6 anywhere else, I didn't think it was worth it to implement a general solution for it.

Here is a screenshot with all headings in non-bold and bold with normal text in between for comparison.

![headings-comparison](https://user-images.githubusercontent.com/12495415/81939320-7bd0db00-95f6-11ea-8ed2-13b985fde96a.png)